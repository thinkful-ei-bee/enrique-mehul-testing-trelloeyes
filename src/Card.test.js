import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';



it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Card  />,
      div,
    );
    ReactDOM.unmountComponentAtNode(div);
   
  });

  it('get back proper card when returned ', () => {
    
    const tree = renderer
      .create(<Card title="3 of Spades" content="correct content" />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });