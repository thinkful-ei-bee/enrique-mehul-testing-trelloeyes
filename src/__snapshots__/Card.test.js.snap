// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get back proper card when returned  1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    3 of Spades
  </h3>
  <p>
    correct content
  </p>
</div>
`;
